### PR TITLE
Remove support for MySQL5.5

### PR DESCRIPTION
**Pull request by sshistrava**

### CODE REVIEW

File: lib/lynx/pipe/append.rb [hunk 1]
```diff
@@ -9,7 +9,8 @@ def initialize(file)
       end
 
       def perform(command)
-        SystemOut.run("#{command} >> #{@file}")
+        output = SystemOut.run(" #{command}")
+        File.open(@file, 'a') { |file| file.write(output) }
```
Comment (panthomakos) [inline lib/lynx/pipe/append.rb]: Should this be a `puts` so that we get newlines?

File: lib/lynx/pipe/append.rb [hunk 2]
```diff
@@ -9,7 +9,8 @@ def initialize(file)
       end
 
       def perform(command)
-        SystemOut.run("#{command} >> #{@file}")
+        output = SystemOut.run(" #{command}")
```
Comment (panthomakos) [inline lib/lynx/pipe/append.rb]: nit: space

File: lib/lynx/pipe/append.rb (hunk 1 above)
Comment (sshistrava) [reply lib/lynx/pipe/append.rb]: Given that everything is working fine now (particularly with Docker), I reverted this change and am just removing support for mysql5.5 with this pr

File: lib/lynx/pipe/append.rb (hunk 2 above)
Comment (sshistrava) [reply lib/lynx/pipe/append.rb]: Given that everything is working fine now (particularly with Docker), I reverted this change and am just removing support for mysql5.5 with this pr

**Comment by sshistrava** [comment]
I'm not particularly worried about publishing a new lynx version or upgrading the one in active, but leaving this PR here in case you do want to

**Pull request closed without merging**